pkg/products/cloudresources/reconciler.go

```diff
@@ -616,3 +616,13 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 
 	return client.Patch(ctx, cfgMap, k8sclient.Merge)
 }
+
+// createSTSARNSecret create the STS arn secret - should be already validated in preflight checks
+func (r *Reconciler) copySTSSecret(ctx context.Context, client k8sclient.Client, operatorNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
+	err := resources.CopySecret(ctx, client, sts.CredsSecretName, operatorNamespace, sts.CredsSecretName, r.Config.GetOperatorNamespace())
```

Comment:
```suggestion
	err := resources.CopySecret(ctx, client, sts.CredsSecretName, operatorNamespace, sts.CredsSecretName, r.ConfigManager.GetOperatorNamespace())
```
Should this be using the `ConfigManager` instead for the rhoam operator namespace ? 🤔

Reply:
no objection to changing it, but at the end of the day the result is the same we get the operator namespace.

Reply:
may be removing this function

Reply:
actually will refactor to check for the secret instead